packages/analytics/shippers/gainsight/README.md

```diff
@@ -0,0 +1,14 @@
+# @kbn/analytics-shippers-gainsight
+
+gainsight implementation as a shipper for the `@kbn/analytics-client`.
+
+## How to use it
+
+This module is intended to be used **on the browser only**. It does not support server-side events.
+
+```typescript
+import { gainsightShipper } from "@kbn/analytics-shippers-gainsight";
+
+analytics.registerShipper(gainsightShipper, { gainsightOrgId: '12345' })
+```
```

Comment:
super-nit: this piece of doc should match the actual usage 😇 

```suggestion
```typescript
import { GainsightShipper } from "@kbn/analytics-shippers-gainsight";

analytics.registerShipper(GainsightShipper, { gainsightOrgId: '12345' })
```
```